tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -156,6 +183,7 @@ def test_standard_bc_creation_and_deletion(
             bc_name, policy, [bs.name for bs in test_stores]
         )
 
```

Comment:
as question was raised upon these comments, adding a clarification:

I usually try to avoid #TODO's, but this time I left them as a reminder to make the test work properly.
The problem why I can not implement TODO's now, is the test require complete rework with a new test flow, locators, etc.
Currently it has @skipif_ocs_version("!=4.8"). Making this test work goes beyound boundaries of this PR

@sagihirshfeld